File: pclassic/stubs-2016/AthenianVoterFraud.py
def athenian_voter_fraud(votes):
    """

    :param votes: list containing the identifier for every ballot which was submitted
    :return: list of identifiers of voters who voted more than once
    """
    #write your code here
   
    

#TOUCH THIS, YOU MUSNT
def parse_file_and_call_function():
    with open("AthenianVoterFraudIN.txt", "r") as f:
        for line in f:
            votes = [int(vote) for vote in line.split(',')]
            out = ','.join(
                [str(vote) for vote in sorted(athenian_voter_fraud(votes))])
            if len(out) > 0:
                print(out)
            else:
                print ('none')


if __name__ == "__main__":
    parse_file_and_call_function()

File: pclassic/stubs-2016/AthenianVoterFraud.java
import java.util.*;
import java.io.*;

class AthenianVoterFraud {
	//AGAIN, DONT TOUCH THE MAIN FUNCTION - IT WILL BE PROVIDED
	public static void main(String[] args)throws IOException {
		
		BufferedReader b = new BufferedReader(new FileReader("AthenianVoterFraudIN.txt"));

		while (b.ready()) {
			String[] data = b.readLine().split(",");
			ArrayList<Integer> votes = new ArrayList<Integer>();
			for (String vote: data) {
				votes.add(Integer.parseInt(vote));
			}
      int[] voters = new int[votes.size()];
      for (int i = 0; i < votes.size(); i++) {
        voters[i] = votes.get(i);
      }
			int[] result = findDuplicates(voters);
      Arrays.sort(result);
			if (result.length == 0) {
				System.out.println("none");
			} else {
				String res = Arrays.toString(result);
				res = res.substring(1, res.length() - 1);
				res = res.replace(", ", ",");
				System.out.println(res);
			}
		}

	}
	
	//THIS IS THE SOLN PROVIDED - THIS IS BAD CODE THIS WASTES A LOT OF SPACE
	public static int[] findDuplicates(int[] voters) {
		//write your code here and change null
		return null;
	}
	
}
